lib/ProductOpener/Food.pm

```diff
@@ -2828,220 +2982,313 @@ fluoride => {
 	en => "Caffeine",
+	cs => "Kofein",
+	de => "Koffein",
 	el => "Καφεΐνη",
+	fr => "Caféine / Théine",
 	nl => "Cafeïne",
 	nl_be => "Cafeïne",
+	nb => "Koffein",
 	pt => "Cafeína",
+	ru => "Кофеин",
+	zh => "咖啡因",
+	zh_CN => "咖啡因",
+	zh_HK => "咖啡因",
+	zh_TW => "咖啡因",
 },
 taurine => {
-	zh_hans => "牛磺酸",
-	zh_hant => "牛磺酸",
-	zh_hk => "牛磺酸",
-	zh_cn => "牛磺酸",
-	zh_sg => "牛磺酸",
-	zh_tw => "牛磺酸",
-	pl => "Tauryna",
-	wa => "Torene",
-	ko => "타우린",
+	ar => "التورين",
+	bg => "Таурин",
+	ca => "Taurina",
+	cs => "Taurin",
+	da => "Taurin",
+	de => "Taurin",
+	el => "Ταυρίνη",
+	en => "Taurine",
+	en_ca => "Taurine",
+	en_gb => "Taurine",
+	eo => "Taŭrino",
+	es => "Taurina",
+	et => "Tauriin",
+	fa => "تائورین",
+	fi => "Tauriini",
 	fr => "Taurine",
+	gl => "Taurina",
 	he => "טאורין",
-	es => "Taurina",
+	hr => "Taurin",
 	hu => "Taurin",
-	it => "Taurina",
-	gl => "Taurina",
-	et => "Tauriin",
+	hy => "Տաուրին",
 	id => "Taurina",
-	de => "Taurin",
+	it => "Taurina",
 	ja => "タウリン",
-	vi => "Taurine",
+	ko => "타우린",
+	mk => "Таурин",
+	nb => "Taurin",
 	nl => "Taurine",
 	nl_be => "Taurine",
-	sv => "Taurin",
+	nb => "Taurin",
+	pl => "Tauryna",
 	pt => "Taurina",
-	eo => "Taŭrino",
+	pt_br => "Taurina",
+	ro => "Taurină",
+	ru => "Таурин",
+	sh => "Taurin",
 	sk => "Taurín",
-	ru => "таурин",
-	en => "Taurine",
-	sr_el => "Taurin",
+	sl => "Tavrin",
+	sq => "taurin",
+	sr => "таурин",
 	sr_ec => "Таурин",
+	sr_el => "Taurin",
+	sv => "Taurin",
 	tr => "Taurin",
-	ro => "Taurină",
-	ca => "Taurina",
-	fi => "Tauriini",
 	uk => "Таурин",
-	sl => "Tavrin",
-	cs => "Taurin",
-	fa => "تائورین",
-	bg => "Таурин",
-	hr => "Taurin",
-	da => "Taurin",
-	sr => "таурин",
+	vi => "Taurine",
+	wa => "Torene",
 	zh => "牛磺酸",
-	nb => "Taurin",
-	hy => "Տաուրին",
-	sh => "Taurin",
-	ar => "التورين",
-	sq => "taurin",
-	mk => "Таурин",
-	el => "Ταυρίνη",
-	en_ca => "Taurine",
-	en_gb => "Taurine",
-	pt_br => "Taurina",
+	zh_CN => "牛磺酸",
+	zh_HK => "牛磺酸",
+	zh_TW => "牛磺酸",
+	zh_cn => "牛磺酸",
+	zh_hans => "牛磺酸",
+	zh_hant => "牛磺酸",
+	zh_hk => "牛磺酸",
+	zh_sg => "牛磺酸",
+	zh_tw => "牛磺酸",
 },
 
 ph => {
-	en => "pH",
+	da => "pH",
+	de => "pH",
 	el => "pH",
+	en => "pH",
+	it => "pH",
+	hu => "pH",
 	nl => "pH",
 	nl_be => "pH",
-	unit => "",
+	nb => "pH",
+	zh => "pH",
+	zh_CN => "pH",
+	zh_HK => "pH",
+	zh_TW => "pH",
+
+	unit => "',
 },
 
 "carbon-footprint" => {
-	fr => "Empreinte carbone / émissions de CO2",
-	en => "Carbon footprint / CO2 emissions",
-	es => "Huella de carbono / Emisiones de CO2",
-	el => "Αποτύπωμα άνθρακα/Εκπομπές CO2",
-	it=> "Emissioni di CO2 (impronta climatica)",
-	pt => "Pegada de carbono / Emissões de CO<sub>2</sub>",
-	de => "Carbon Footprint / CO2-Emissionen",
+	en => "Carbon footprint / CO<sub>2</sub> emissions",
+	cs => "Uhlíková stopa / Emise CO<sub>2</sub>",
+	de => "Carbon Footprint / CO<sub>2</sub>-Emissionen",
+	el => "Αποτύπωμα άνθρακα/Εκπομπές CO<sub>2</sub>",
+	es => "Huella de carbono / Emisiones de CO<sub>2</sub>",
```

Comment:
<pre>Illegal declaration of anonymous subroutine at /home/travis/build/openfoodfacts/openfoodfacts-server/lib/ProductOpener/Food.pm line 3272.

Reply:
[Sat Nov 10 17:09:39 2018] Food.pm: Bareword found where operator expected at /home/travis/build/openfoodfacts/openfoodfacts-server/lib/ProductOpener/Food.pm line 3267, near ""carbon"
[Sat Nov 10 17:09:39 2018] Food.pm:   (Might be a runaway multi-line "" string starting on line 3264)